vocode/streaming/streaming_conversation.py

```diff
@@ -589,6 +617,39 @@ def is_interrupt(self, transcription: Transcription):
         return transcription.confidence >= (
             self.transcriber.get_transcriber_config().min_interrupt_confidence or 0
         )
+    
+
+
+    async def send_noise(self):
+        filler_audio_noise = FillerAudio(
+                        BaseMessage(text="Noise"),
+                        audio_data=convert_wav(
+                            '/Users/mohammadaminroohi/miniconda3/envs/vocode_1/lib/python3.9/site-packages/vocode/streaming/pink BL.wav',
+                            output_sample_rate=8000,
+                            output_encoding=AudioEncoding.MULAW,
+                        ),
+                        synthesizer_config=ElevenLabsSynthesizerConfig(sampling_rate=8000, audio_encoding=AudioEncoding.MULAW),
```

Comment:
This is referencing a file that is not present in the source. Consider adding it to this PR.

Reply:
Yeah done thanks for letting me know. Actually I was about to add background noise to the conversation here. The goal was to make the conversation more natural. But approach is not working since on the same time, either the noise is been sent to Twilio or the synthesized voice, not both simultaneously. Which make the conversation wired.